bridge_adaptivity/module/forms.py

```diff
@@ -30,7 +30,10 @@ class Meta:
         model = Activity
         exclude = ['collection', 'points']
         widgets = {
-            'stype': forms.HiddenInput(), 'points': forms.HiddenInput(), 'lti_lms_platform': forms.HiddenInput()
+            'stype': forms.HiddenInput(),
```

Comment:
Why do we need hidden inputs in the form?

Reply:
Why does lti_lms_platform is removed from hidden input?